integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/api/CrowdNodeApi.kt

```diff
@@ -491,7 +494,11 @@ class CrowdNodeApiAggregator @Inject constructor(
         fullSignUpSet?.let { set ->
             accountAddress = set.accountAddress
             requireNotNull(accountAddress) { "Restored signup tx set but address is null" }
-            configScope.launch { globalConfig.crowdNodeAccountAddress = accountAddress!!.toBase58() }
+            configScope.launch {
+                globalConfig.crowdNodeAccountAddress = accountAddress!!.toBase58()
+                transactionMetadataProvider.maybeMarkAddressWithTaxCategory(accountAddress!!.toBase58(), false, TaxCategory.TransferIn)
+                transactionMetadataProvider.maybeMarkAddressWithTaxCategory(accountAddress!!.toBase58(), true, TaxCategory.TransferOut)
+            }
```

Comment:
anytime that the `crowdNodeAccountAddress` is saved to the configuration, this address is saved to AddressMetadata table.

Reply:
3 places were found where this happens, so each was marked.